packages/web/docs/src/pages/product-updates/2024-07-30-persisted-documents-app-deployments-preview.mdx

```diff
@@ -0,0 +1,99 @@
+---
+title: Persisted Documents with App Deployments in Preview
+description:
+  Secure your GraphQL API and reduce upstream traffic by persisting documents through app
+  deployments.
+date: 2024-07-30
+authors: [laurin]
+---
+
+import NextImage from 'next/image'
+
+**TL;DR** Persisted documents through app deployments are now available in preview on Hive and allow
+you to secure your GraphQL API. To get started, please refer to the
+[app deployments documentation](/docs/features/app-deployments).
+
+## What are Persisted Documents?
+
+Persisted documents (also known as persisted queries or trusted documents) are a way to pre-register
```

Comment:
should we link to some more docs on these terms?
```suggestion
Persisted documents (also known as [persisted queries](https://relay.dev/docs/guides/persisted-queries/) or [trusted documents](https://benjie.dev/graphql/trusted-documents)) are a way to pre-register
```

Reply:
I would not link to relay docs, they don't provide anything useful information on why